cve_bin_tool/output_engine/__init__.py

```diff
@@ -190,9 +207,17 @@ def output_cves(self, outfile, output_type="console"):
         else:  # console, or anything else that is unrecognised
             output_console(self.all_cve_data)
 
+        if self.append_flag is True and self.intermediate_file is not None:
```

Comment:
Hm, `if self.append_flag is True`? I thought it is idiomatic to just do `if self.append_flag`
```suggestion
        if self.append_flag and self.intermediate_file is not None:
```